src/app/shared/table/table.component.ts

```diff
@@ -134,6 +136,29 @@ export class TableComponent implements OnInit, AfterViewInit, OnDestroy {
     this.refresh();
   }
 
+  public dateRangeChanged(filterDef: TableFilterDef, event: any){
```

Comment:
[low] White space (WS) before the curly brace (install eslint in VSCode)

Reply:
ESLint doesn't seem to be throwing errors for these on my machine.